server/build-plugins.js

```diff
@@ -95,9 +94,17 @@ const prepareModules = async (plugins = [], local = true) => {
 
 (async () => {
   try {
+    assert(
+      fs.existsSync(pluginsConfig),
+      'bPanel config file not found. Please run `npm install` before \
```

Comment:
These types of messages could be constants

Reply:
Where else would we use this specific message? Maybe related but I definitely want to hook into the Chrome notifications API to handle error and other messaging.